pkg/apis/certman/v1alpha1/certificaterequest_types.go

```diff
@@ -40,16 +40,11 @@ type CertificateRequestSpec struct {
 	DnsNames []string `json:"dnsNames"`
 
 	// Let's Encrypt will use this to contact you about expiring certificates, and issues related to your account.
-	// +optional
 	Email string `json:"email"`
```

Comment:
Hmm, the comment says this is optional, but the `Email` field will show up in json, even if it's unused, since it doesn't have  `omitempty`. Is this intentional? It is ok to have the field blank when it's marshalled into json? Like `Email: ""`

Reply:
With this commit, it is no longer optional. The diff is removing the `optional` annotation.